assets/react/controllers/Mosaic.jsx

```diff
@@ -180,6 +180,7 @@ function Mosaic() {
                                     exposed
                                     tileIcons={config.variant.exposeShowIcon ?? false}
                                     tileBorders={config.variant.exposeShowBorder ?? false}
+                                    exposeFontSize={config.variant.exposeFontSize}
```

Comment:
What is the default value for this?
If you do not set `config.variant.exposeFontSize`, then what happens?

Reply:
Thanks @tuj - I have added a fallback value